quiche/src/h3/mod.rs

```diff
@@ -1280,6 +1296,17 @@ impl Connection {
             conn.dgram_max_writable_len().is_some()
     }
 
+    /// Returns whether the peer enabled HTTP/3 CONNECT protocol support.
```

Comment:
for consistency with enable_extended_connect, how about
```suggestion
    /// Returns whether the peer enabled extended CONNECT support.
```